test/unit/org/apache/cassandra/db/guardrails/GuardrailsTest.java

```diff
@@ -63,10 +63,10 @@ private void testDisabledThreshold(Threshold guard) throws Throwable
     @Test
     public void testThreshold() throws Throwable
     {
-        Threshold guard = new Threshold("x",
+        MaxThreshold guard = new MaxThreshold("x",
```

Comment:
We should probably add some test for `MinThreshold`. We could rename `testThreshold` to `testMaxThreshold`, and add a new analogous `testMinThreshold`.